### ISSUE TITLE
NTR: 4,5-dihydrohexanoic acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8625]
New term request 4,5-dihydrohexanoic acid high in urine (segal_040303233833):

**Comment by pnrobinson** [comment]
@MickeySegal
I am having trouble finding information about this compound. Can you specify?

**Comment by MickeySegal** [comment]
https://pubmed.ncbi.nlm.nih.gov/12829002/ "4,5-Dihydrohexanoate, 2-deoxytetronate and 3-deoxytetronate were also 1.7-2.7-fold higher."

**Comment by pnrobinson** [comment]
@MickeySegal 
I am unable to find this designation in CHEBI or other databases of biochemicals and am wondering if it is imprecise or wrong.

**Comment by MickeySegal** [comment]
You are right.  Two sources had the same typo.  The correct name is 4,5-dihydroxyhexanoic acid

https://hmdb.ca/metabolites/HMDB0240745
4,5-dihydroxyhexanoic acid or DHHA is a water-soluble organic acid. It probably arises from the reaction of succinic semialdehyde with an intermediate in the pyruvate dehydrogenase pathway. DHHA is found in the urine and appears to be a marker for Succinic semialdehyde dehydrogenase (SSADH) deficiency (PMID: [3126356 ](http://www.ncbi.nlm.nih.gov/pubmed/3126356)).

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001003
- label: Elevated urine 4,5-dihydroxyhexanoic acid level
- parents: [Elevated urinary carboxylic acid (HP:0040156)](https://hpo.jax.org/app/browse/term/HP:0040156)
- pmid: 
- def: The amount of 4,5-dihydroxyhexanoic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8625
- synonyms:
